### PR TITLE
Nano pico priorities

### PR DESCRIPTION
**Pull request by Dekabry**
## Describe your PR

Added pico & nano callout mentionning they have lower priorities (and explaining why sometimes performance might drop).
Added a link from "Quickstart" to "Scalability" to better inform new users about instance sizes.

## Checklist

- [x] My PR is related to an opened issue : #48
- [x] I've read the [contributing guidelines](/CleverCloud/documentation/blob/main/CONTRIBUTING.md)

## Reviewers
@CleverCloud/reviewers

### CODE REVIEW

**Comment by Dekabry** [comment]
@cnivolle  Nope, that was an accident, i pushed that one on another PR and i'm not sure why it pushed it it here as well.
I think i should have deleted my `haskell-variables` branch ?

**Comment by cnivolle** [comment]
@Dekabry the callout should be above "Enable auto-scalability" for an easy reading.

**Comment by Kirbeerus** [comment]
@Dekabry I just took a look at the review app, and i don't see the callout about Nano and Pico appear on the page `Application Scaling`.
Can you check if you also have the problem.

**Comment by Dekabry** [comment]
@Kirbeerus  i see it right here from the review app ? 
image

File: content/doc/quickstart/_index.md [hunk 1]
```diff
@@ -116,6 +116,11 @@ Choose the language or the framework you want to deploy.
 
 Horizontal scaling is the number of instances that can run at the same time. Vertical scaling sets the minimum and maximum size the instance can be.
 
+{{< callout emoji="💡" >}}
+  You can learn more about scaling & instances size [here](https://developers.clever-cloud.com/doc/administrate/scalability/).
```
Comment (Kirbeerus) [inline content/doc/quickstart/_index.md:120]:
```suggestion
  You can learn more about scaling & instances size [here](/doc/administrate/scalability).
```

Comment (Kirbeerus) [reply content/doc/quickstart/_index.md:120]: Instead of using a link that directly send to the real documentation, use a reference to the doc.

**Review by Kirbeerus — CHANGES_REQUESTED** [review]

**Review by Kirbeerus — APPROVED** [review]

**Pull request merged**